### PR TITLE
IntMax Validator

### PR DESCRIPTION
**Pull request by LuizMoratelli**
int-max-validator

### CODE REVIEW

File: packages/bs-reform/src/ReFormNext.re [hunk 1]
```diff
@@ -38,7 +40,11 @@ module Make = (Config: Config) => {
     switch (validator) {
     | Validation.IntMin(field, min) => (
         Field(field),
-        Config.get(values, field) > min ? Valid : Error("Below minimum"),
+        Config.get(values, field) >= min ? Valid : Error("Below minimum"),
```
Comment (yurijean) [inline packages/bs-reform/src/ReFormNext.re]: You can use Yup's error message, which include the `min` value https://github.com/jquense/yup/blob/master/src/locale.js#L38

File: packages/bs-reform/src/ReFormNext.re [hunk 2]
```diff
@@ -38,7 +40,11 @@ module Make = (Config: Config) => {
...
+      )
+    | Validation.IntMax(field, max) => (
+        Field(field),
+        Config.get(values, field) <= max ? Valid : Error("Above maximum"),
```
Comment (yurijean) [inline packages/bs-reform/src/ReFormNext.re]: Here you can use Yup's error message again

File: packages/bs-reform/src/ReFormNext.re (hunk 1 above)
Comment (fakenickels) [reply packages/bs-reform/src/ReFormNext.re]: noice, really soon we are going to extract all these and put in a real errors dictionary so users are able to inject their own.

File: packages/bs-reform/src/ReFormNext.re [hunk 3]
```diff
@@ -38,7 +40,21 @@ module Make = (Config: Config) => {
     switch (validator) {
     | Validation.IntMin(field, min) => (
         Field(field),
-        Config.get(values, field) > min ? Valid : Error("Below minimum"),
+        Config.get(values, field) >= min
+          ? Valid
+          : Error(
+              "This value must be greater than or equal to "
+              ++ string_of_int(min),
```
Comment (yurijean) [inline packages/bs-reform/src/ReFormNext.re]: :shipit:

**Comment by guilhermedecampo** [comment]
haha I loved the spinning landing page! hasiuhiusa

**Comment by fakenickels** [comment]
:sanic:

**Pull request merged**